yt/visualization/volume_rendering/scene.py

```diff
@@ -437,15 +444,46 @@ def save_annotated(
         """
         fname = self._setup_save(fname, render)
 
-        # which transfer function?
-        rs = self._get_render_sources()[0]
-        tf = rs.transfer_function
-        label = rs.data_source.ds._get_field_info(rs.field).get_label()
-
         ax = self._show_mpl(
             self._last_render.swapaxes(0, 1), sigma_clip=sigma_clip, dpi=dpi
         )
-        self._annotate(ax.axes, tf, rs, label=label, label_fmt=label_fmt)
+
+        # number of transfer functions?
+        num_trans_func = 0
+        for rs in self._get_render_sources():
+            if hasattr(rs, "transfer_function"):
+                num_trans_func += 1
+
+        # which transfer function?
+        if num_trans_func == 1:
+            rs = self._get_render_sources()[0]
+            tf = rs.transfer_function
+            label = rs.data_source.ds._get_field_info(rs.field).get_label()
+            self._annotate(ax.axes, tf, rs, label=label, label_fmt=label_fmt)
+        else:
+            # set the origin and width and height of the colorbar region
+            if tf_rect is not None:
+                cbx0, cby0, cbw, cbh = tf_rect
+
+            if tf_rect is None:
+                cbx0 = 0.80
+                cby0 = 0.12
+                cbw = 0.12
+                cbh = 0.9
```

Comment:
Oh, sorry I didn't see this earlier. What do you think about this simplification ?
```suggestion
            if tf_rect is None:
                tf_rect = 0.80, 0.12, 0.12, 0.9
            cbx0, cby0, cbw, cbh = tf_rect
```

Reply:
Alternatively it could even be done in one line
```suggestion
            cbx0, cby0, cbw, cbh = tf_rect or (0.80, 0.12, 0.12, 0.9)
```

Reply:
I like the 3 line version better.  It's more readable.